### PR TITLE
Update the Lira config template to support running Adapter Optimus.

### PR DESCRIPTION
**Pull request by rexwangcc**
### Purpose
_Please explain the purpose of this PR and include links to any GitHub issues that it fixes:_

- https://app.zenhub.com/workspaces/dcp-backlogs-5ac7bcf9465cb172b77760d9/issues/humancellatlas/secondary-analysis/368
---
### Changes
_Please list out what major changes were made in this PR to address the issue:_

- Update the Lira config template to support running Adapter Optimus.
---
### Review Instructions
_Please provide instructions about how should a reviewer test/verify the changes in this PR:_

- I will kick off a series of workflows manually and paste the list here for review.

---
### PR Checklist
_Please ensure the following when opening a PR:_

- [ ] This PR added or updated tests.
- [ ] This PR updated docstrings or documentation.
- [ ] This PR applied Python style guidelines, specifically followed [Google style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google) for this repo.
- [ ] This PR considered generalizability beyond our own use case.

---
### Follow-up Discussions
_Please append follow-up discussions and issues during the review process below:_

- No follow-up discussions.

### CODE REVIEW

File: deploy/config_files/config.sh.ctmpl [hunk 1]
```diff
@@ -167,4 +179,14 @@
     export TENX_WDL_STATIC_INPUTS_LINK="${PIPELINE_TOOLS_PREFIX}/adapter_pipelines/cellranger/adapter_example_static.json"
     export TENX_WDL_LINK="${PIPELINE_TOOLS_PREFIX}/adapter_pipelines/cellranger/adapter.wdl"
     export TENX_WORKFLOW_NAME="Adapter10xCount"
+
+    # Optimus Variables
+    export OPTIMUS_PREFIX="https://raw.githubusercontent.com/HumanCellAtlas/skylab/${OPTIMUS_VERSION}"
+    export OPTIMUS_ANALYSIS_WDLS="[
+                    \"${OPTIMUS_PREFIX}/pipelines/optimus/Optimus.wdl\"
```
Comment (samanehsan) [inline deploy/config_files/config.sh.ctmpl]: Please include the WDL files that Optimus imports as well: https://github.com/HumanCellAtlas/skylab/blob/master/pipelines/optimus/Optimus.wdl#L1-L12

**Comment by samanehsan** [comment]
Please update the mintegration test in secondary-analysis to include these new variables for Optimus! 🌊

File: deploy/config_files/config.sh.ctmpl (hunk 1 above)
Comment (rexwangcc) [reply deploy/config_files/config.sh.ctmpl]: Oops, good catch!

**Review by samanehsan — APPROVED** [review]

**Comment by rexwangcc** [comment]
Close by https://github.com/HumanCellAtlas/secondary-analysis-deploy/pull/52

**Pull request closed without merging**